Mathlib/CategoryTheory/Monoidal/Cartesian/FunctorCategory.lean

```diff
@@ -167,6 +170,29 @@ instance {K : Type*} [Category K] [HasColimitsOfShape K C]
     NatIso.ofComponents (fun _ ↦ Iso.refl _)
   exact preservesColimitsOfShape_of_natIso this.symm
 
+/-- A finite-products-preserving functor distributes over the tensor product of functors. -/
+noncomputable def tensorObjComp (F G : D ⥤ C) (H : C ⥤ E) [PreservesFiniteProducts H] :
+    (F ⊗ G) ⋙ H ≅ (F ⋙ H) ⊗ (G ⋙ H) :=
+  NatIso.ofComponents (fun X ↦ prodComparisonIso H (F.obj X) (G.obj X)) fun {X Y} f ↦ by
+    dsimp; ext <;> simp [← Functor.map_comp]
```

Comment:
As (the hom direction of) this is a map to a limit, it needs to be characterized w.r.t the projections. things like 
```
@[reassoc (attr := simp)]
lemma tensorObjComp_hom_fst_app (F G : D ⥤ C) (H : C ⥤ E) [PreservesFiniteProducts H] (x : D) :
    (tensorObjComp _ _ F G H).hom.app x ≫ (fst (F ⋙ H) (G ⋙ H)).app x = H.map (fst _ _) := by 
  simp [tensorObjComp]
```

Reply:
I've added `simps` instead, is that okay by you?